chia/wallet/singleton.py

```diff
@@ -14,7 +17,7 @@
 SINGLETON_LAUNCHER_PUZZLE_HASH = SINGLETON_LAUNCHER_PUZZLE.get_tree_hash()
 
 
-def get_inner_puzzle_from_singleton(puzzle: Program) -> Optional[Program]:
+def get_inner_puzzle_from_singleton(puzzle: Union[Program, SerializedProgram]) -> Optional[Program]:
```

Comment:
I think I would prefer `Program.to`ing the call sites instead but idc that much

Reply:
left as-is for now